bindings/java/c/host.h

```diff
@@ -11,11 +11,6 @@
 extern "C" {
 #endif
 
-struct evmc_host_context
```

Comment:
The whole point of `evmc_host_context` is to be an opaque object from the interface perspective, and since it only needs to be valid during the scope of `execute` I think this change is the correct one.